ui/packages/shared/components/src/Select/index.tsx

```diff
@@ -69,101 +69,92 @@ const Select = ({
-                      <div className="w-[270px]">{loader}</div>
-                    </div>
-                  ) : (
-                    <>
-                      {items.length > 0 && (
-                        <div
-                          className={cx(
-                            width !== undefined ? `w-${width}` : '',
-                            'absolute z-10 mt-1 bg-gray-50 dark:bg-gray-900 dark:border-gray-600 shadow-lg rounded-md py-1 text-base ring-1 ring-black dark:ring-white ring-opacity-5 dark:ring-opacity-20 overflow-auto focus:outline-none sm:text-sm'
-                          )}
-                        >
-                          {items.map(option => (
-                            <Listbox.Option
-                              key={option.key}
-                              disabled={option.disabled ?? false}
-                              className={({active, disabled}) =>
-                                cx(
-                                  active && 'text-white bg-indigo-600',
-                                  'cursor-default select-none relative py-2 pl-3 pr-9',
-                                  disabled && 'opacity-50'
-                                )
-                              }
-                              value={option.key}
-                            >
-                              {({selected, active}) => (
-                                <>
-                                  <div className="flex items-center">
-                                    <span
-                                      className={cx(
-                                        selected ? 'font-semibold' : 'font-normal',
-                                        'ml-3'
-                                      )}
-                                    >
-                                      {option.element.expanded}
-                                    </span>
-                                  </div>
-                                  {selected ? (
-                                    <span
-                                      className={cx(
-                                        active ? 'text-white' : 'text-indigo-600',
-                                        'absolute inset-y-0 right-0 flex items-center pr-4'
-                                      )}
-                                    >
-                                      <Icon icon="heroicons:check-20-solid" aria-hidden="true" />
-                                    </span>
-                                  ) : null}
-                                </>
-                              )}
-                            </Listbox.Option>
-                          ))}
-                        </div>
+    <Listbox value={selectedKey} onChange={onSelection}>
+      {({open}) => (
+        <div className="min-w-fit relative">
```

Comment:
Other than cosmetic changes introduced to the code (for example, removing React fragment elements <></> where not necessary) this is the only significant change introduced in this PR.